app.js

```diff
@@ -1,9 +1,26 @@
 const express = require("express");
 const app = express()
-const { getTopics, getEndPoints } = require('./controllers/topics-controller')
+const { getTopics } = require('./controllers/topics-controller')
+const { getArticleById } = require('./controllers/articles-controller')
+const { getEndPoints } = require('./controllers/endpoints-controller')
 
 app.get("/api/topics", getTopics);
-console.log(getEndPoints)
+
+app.get("/api/articles/:article_id", getArticleById);
+
 app.get("/api", getEndPoints);
 
+app.use((err, req, res, next) => {
+    if (err.msg === 'Not found'){
+      res.status(404).send({msg: err.msg})
+    }
+    else next(err)
+  })
+
+  app.use((err, req, res, next) => {
+    if (err.msg === 'Bad request'){
+      res.status(400).send({msg: err.msg})
+    }
+  })
+
```

Comment:
See comment above, this may help reduce lines of code for essentially the same thing